docs/reference/ml/anomaly-detection/ml-configuring-aggregations.asciidoc

```diff
@@ -45,6 +45,18 @@ finer, more granular time buckets, which are ideal for this type of analysis. If
 your detectors use <<ml-count-functions,count>> or <<ml-rare-functions,rare>>
 functions, set the interval to the same value as the bucket span.
 
+If your <<aggs-dfeeds,{dfeed} uses aggregations with nested `terms`>> and model 
+plot is not enabled for the {anomaly-job}, then neither can the **Single Metric 
+Viewer** nor the **Anomaly Explorer** in {kib} plot and display an anomaly 
+chart for the job.
+
+When the aggregation interval of the {dfeed} and the bucket span of the job 
+don't match, the values of the chart plotted in both the Single Metric Viewer 
+and the Anomaly Explorer differs from the actual values of the job. To avoid 
```

Comment:
Since I think it's the the "values" (plural) that differ, we should drop the "s" here:
```suggestion
and the **Anomaly Explorer** differ from the actual values of the job. To avoid 
```